### ISSUE TITLE
Cannot buffer large files (video etc.) while downloading

### ISSUE THREAD

**Issue by libersoft-org** [elysiajs/elysia#300]
I have a problem with serving a static file through Elysia like this:

```javascript
app.get('/download/:hash/:name', async (req) =>{
 const file = Bun.file(path.join(dirDownload, req.params.hash));
 new Response(file, {
  headers: {
   'Content-Type': 'application/octet-stream',
   'Content-Disposition': 'attachment; filename="' + req.params.name + '"'
  }
 });
});
```
I cannot play big video files (in "video" HTML tag on frontend) before it's 100% of the file buffered (completely downloaded) - this does not happen in Express.js using res.sendFile, only in Bun + Elysia... Any idea on how to buffer big files? There is nothing like this in Elysia documentation.

**Comment by bogeychan** [comment]
relates to https://github.com/elysiajs/elysia-static/issues/19

**Comment by SaltyAom** [comment]
Fixed by 4ba3e85 published under 0.8.0.

Feel free to reopen the issue if the problem still persists.
